### ISSUE TITLE
Login: Multiple People on Browser

### ISSUE THREAD

**Issue by MappingKat** [MappingKat/issues#5]
Work/Personal Chrome browser.. how does that work on login?

**Comment by imcmahon** [comment]
You can login on multiple computers at once with the normal email login process. Does that answer the question?

Multiple people using a computer would need to logout and in. That's outside of our base use case for right now though.

**Comment by MappingKat** [comment]
Yes.  It was a thought I had that might come up in the future.  We can keep it dormant.
